authserver/api/static/css/style.css

```diff
@@ -1,51 +1,80 @@
-html,
-body {
-  height: 100%;
+:root {
+  --blue: #0fbaa8;
+  --light-blue: #0fbaa8b3;
+  --gray20: #333333;
 }
 
 body {
-  display: -ms-flexbox;
-  display: -webkit-box;
+  font-family: "Montserrat", sans-serif;
+  font-size: 18px;
+}
+
+.login {
   display: flex;
-  padding-bottom: 40px;
-  background-color: #333333;
-  font-family: "Raleway", sans-serif;
+  /* "column" – major axis is vertical */
```

Comment:
forgot to remove the comments?

Reply:
oopps

Reply:
https://github.com/brighthive/authserver/pull/40/commits/402262741751ad409213e32bc77e85685a1a1844